bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -2494,6 +2500,14 @@ CGINS-SPKIRJ-00265,N00560,Sensor,0,Spectral Irradiance: SPKIR Series J,SPKIR for
 CGINS-SPKIRJ-00266,N00576,Sensor,0,Spectral Irradiance: SPKIR Series J,SPKIR for WLP-003,Satlantic,OCR507 ICSW w/ Bioshutter II (Midrange),266,,20140616,13316.36,,Oregon State University
 CGINS-SPKIRJ-00281,,Sensor,0,Spectral Irradiance: SPKIR Series J,SPKIR for WLP-007,Satlantic,OCR507 ICSW w/ Bioshutter II (Midrange),281,,20161230,,,Oregon State University
 CGINS-SPKIRJ-00282,,Sensor,0,Spectral Irradiance: SPKIR Series J,SPKIR for WLP-008,Satlantic,OCR507 ICSW w/ Bioshutter II (Midrange),282,,20161230,,,Oregon State University
+CGINS-TURBDX-01106,A00424,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1106,,20130920,12634.25,same instrument as CGINS-FLORTD-01106 with 0-250 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01213,A00888,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1213,,20140627,13013.52,same instrument as CGINS-FLORTD-01213 with 0-3 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01214,A00889,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1214,,20140627,13013.52,same instrument as CGINS-FLORTD-01214 with 0-3 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01215,A01496,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1215,,20140911,13013.52,same instrument as CGINS-FLORTD-01215 with 0-3 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01216,A01497,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1216,,20150911,13013.52,same instrument as CGINS-FLORTD-01216 with 0-3 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01218,A00993,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1218,,20140821,13013.52,same instrument as CGINS-FLORTD-01218 with 0-250 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01295,A01340,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1295,,20150504,13403.46,same instrument as CGINS-FLORTD-01295 with 0-200 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01320,A01633,Sensor,0,3-Wavelength Fluorometer: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1320,,20150717,13403.46,same instrument as CGINS-FLORTD-01320 with 0-250 NTU cal,Woods Hole Oceanographic Institution
```

Comment:
I think we should use "Turbidity Sensor: TURBD Series X" instead of "3-Wavelength Fluorometer: TURBD Series X" for the DESCRIPTION OF EQUIPMENT column.